portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,33 +20,86 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.PreparedQuery;
 
 import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+import com.google.sps.data.Comment; // TO BE EDITED FOR COMMENTS CLASS
+import java.util.List;
+
+
+/** Servlet that  handle and return comments data modify */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
   @Override
   //public static void main(String[] args) { 
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    PreparedQuery results = datastore.prepare(query);
+
+    //response.setContentType("text/plain");
```

Comment:
Commented code?

Reply:
ah! sorry i missed this whilst deleting my commented code/.